bin/release

```diff
@@ -0,0 +1,160 @@
+    run "git checkout -b stages/int origin/stages/int" unless DRY_RUN
+  end
+
+  unless `git branch`.include?(RC_BRANCH)
+    puts "==> Checking out RC branch for release to prod 3 days from today"
+    run "git checkout -b #{RC_BRANCH}" unless DRY_RUN
+
+    puts "==> Merging master into RC branch"
+    run "git merge master" unless DRY_RUN
+
+    puts "==> Pushing RC branch to GitHub"
+    run "git push origin #{RC_BRANCH}" unless DRY_RUN
+  end
+end
+
+def remove_login_dot_gov_dir
+  Dir.chdir ENV['HOME'] do
+    puts "===> Removing ~/login-dot-gov"
+    run "rm -rf login-dot-gov"
+  end
+end
+
+def publish_wiki_page_for_new_rc
+  Dir.chdir ENV['HOME'] do
+    puts "==> Cloning identity-private.wiki repo into ~/login-dot-gov/identity-private.wiki"
+    run "mkdir login-dot-gov"
+    Dir.chdir "#{ENV['HOME']}/login-dot-gov" do
+      run "git clone git@github.com:18F/identity-private.wiki"
+    end
+  end
+
+  Dir.chdir "#{ENV['HOME']}/login-dot-gov/identity-private.wiki" do
+    rc_commit = `git log --grep="#{RC_BRANCH}" --oneline`
+    unless rc_commit.include?(RC_BRANCH)
+      puts "==> Generating the wiki page for the RC"
+      run "./scripts/generate-release --release-date=#{RELEASE_DATE}" unless DRY_RUN
+      puts "==> Pushing the wiki page to GitHub"
+      unless DRY_RUN
+        run "git add ."
+        run "git commit -m \"Add page for #{RC_BRANCH}\""
+        run "git push origin master"
+      end
+    end
+  end
+
+  remove_login_dot_gov_dir
+end
+
+def clone_identity_devops_repo
+  Dir.chdir ENV['HOME'] do
+    puts "==> Cloning identity-devops repo into ~/login-dot-gov/identity-devops"
+    run "mkdir login-dot-gov"
+    Dir.chdir "#{ENV['HOME']}/login-dot-gov" do
+      run "git clone git@github.com:18F/identity-devops.git"
+    end
+  end
+end
+
+def deploy_to_int
+  clone_identity_devops_repo
+
+  Dir.chdir "#{ENV['HOME']}/login-dot-gov/identity-devops" do
+    puts "Deploying the `stages/int` branch to the `int` environment"
+    ENV['AWS_PROFILE'] = 'identitysandbox.gov'
+    run "bin/asg-recycle.sh int idp"
+    run "bin/asg-recycle.sh int worker"
+  end
+
+  remove_login_dot_gov_dir
+end
+
+def deploy_to_staging
+  clone_identity_devops_repo
+
+  Dir.chdir "#{ENV['HOME']}/login-dot-gov/identity-devops" do
+    puts "Deploying the `stages/staging` branch to the `staging` environment"
+    ENV['AWS_PROFILE'] = 'login.gov'
+    run "bin/asg-recycle.sh staging idp"
+    run "bin/asg-recycle.sh staging worker"
+  end
+
+  remove_login_dot_gov_dir
+end
+
+def deploy_to_prod
+  clone_identity_devops_repo
+
+  Dir.chdir "#{ENV['HOME']}/login-dot-gov/identity-devops" do
+    puts "Deploying the `stages/prod` branch to the `prod` environment"
+    ENV['AWS_PROFILE'] = 'login.gov'
+    run "bin/asg-recycle.sh prod idp"
+    run "bin/asg-recycle.sh prod worker"
+  end
+
+  remove_login_dot_gov_dir
+end
+
+def open_pr_for_int
+  run "brew install hub" unless `brew list -1 | grep -Fqx hub`
```

Comment:
FYI, this did not actually install hub, presumably because it exited with a zero status code?